docs/monitoring/servers.md

```diff
@@ -15,6 +15,8 @@ Sematext Agent collects a plethora of metrics about hosts (CPU, memory, disk, ne
 
 ![Infra server metrics](../images/monitoring/infra-server-metrics.gif)
 
+Metrics from both Linux and Windows servers can be collected. Learn more about [Linux Monitoring](https://sematext.com/docs/integration/linux/) and [Windows Monitoring](https://sematext.com/docs/integration/windows/) at Sematext.
```

Comment:
Maybe "Both Linux and Windows servers can be monitored" because our monitoring is not just about L & W metrics?

Reply:
This paragraph is about server metrics, which is why I'm referring to metrics collection only. In the next sentence, I'm generalizing this by mentioning Linux and Windows Monitoring.